### DISCUSSION TITLE
Create new issue on filtered view should use settings to make it visible on that view?

### DISCUSSION THREAD

**Discussion by drone1** [community/community#11880] [Projects and Issues]
Or at least have some options there to set defaults for a new item on a particular view. 

I created a view for my next milestone and added an item there, expected it to be for my proj w/ my milestone, since that's all that's visible. I understand this isn't always desirable but there needs to be a way to configure this so tasks can be created quickly!

Thanks.

**Comment by bwells78** [comment]
Came here to say this, found many others already doing so-
